src/ekore/anomalous_dimensions/unpolarized/space_like/as4/__init__.py

```diff
@@ -49,9 +51,12 @@ def gamma_singlet(N, nf, sx):
     gamma_gg : :math:`\gamma_{gg}^{(3)}`
 
     """
-    gamma_qq = gamma_nsp(N, nf, sx) + gamma_ps(N, nf, sx)
+    gamma_qq = gamma_nsp(N, nf, sx) + gamma_ps(N, nf, sx, variation[3])
```

Comment:
Would be possible to make the fact that `variation[3]` is related to `gamma_ps` more explicit? (and same for the others)

Reply:
I agree, maybe just add a comment.